index.html

```diff
@@ -62,7 +62,7 @@ <h3>
                 <a id="baltoro-climbing-centre" class="anchor" href="#baltoro-climbing-centre" aria-hidden="true"><span
                             class="octicon octicon-link"></span></a>Baltoro Climbing Centre</h3>
 
-            <p>Baltoro climbing centre is the biggest climbing wall in Rzeszów. Here you can find <a
+            <p>Baltoro climbing centre is the biggest boulder climbing wall in Rzeszów. Here you can find <a
```

Comment:
I'm not sure if there shouldn't be coma sign between words boulder and climbing.

Reply:
@jhryniuk asked specialist and it's not necessary.